pyside6_overlay/ui/workspace_manager.py

```
@@ -0,0 +1,158 @@
+            ('fishing_stats', 1, 1)
+        ]
+        
+        for panel_id, row, col in panels:
+            panel = self.panel_factory.create_panel(panel_id)
+            self.active_panels[f'panel_{row}_{col}'] = panel
+            layout.addWidget(panel, row, col)
+            
+    def update_panel_data(self, panel_type: str, data: dict):
+        """Update panel with new data"""
+        # Find panels that need updating
+        for panel_key, panel in self.active_panels.items():
```

Ruff (0.11.9)

Loop control variable `panel_key` not used within loop body

Rename unused `panel_key` to `_panel_key`

(B007)